glean-core/megazord.uniffi.toml

```diff
@@ -0,0 +1,6 @@
+[bindings.swift]
+cdylib_name = "glean_ffi"
```

Comment:
Is this actually still correct?

Reply:
Oops, no...   while it doesn't make a difference whether this works or not this line isn't necessary, it works with or without it.  I intended to remove it but forgot.